### ISSUE TITLE
Unit tests of CondTools/ECAL failing in CMSSW_12_0_X_2021-07-19-1100

### ISSUE THREAD

**Issue by iarspider** [cms-sw/cmssw#34551]
Log file: https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc900/CMSSW_12_0_X_2021-07-19-1100/unitTestLogs/CondTools/Ecal#/45

Failing tests: EcalDAQ_O2O_test, EcalDCS_O2O_test, EcalLaser_O2O_test

Error message:
```
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/week0/slc7_amd64_gcc900/cms/cmssw/CMSSW_12_0_X_2021-07-19-1100/bin/slc7_amd64_gcc900/popconRun", line 7, in <module>
    from CondCore.Utilities import popcon2dropbox
ImportError: No module named CondCore.Utilities
``` 

Most likely caused by dropping python2.

**Comment by makortel** [comment]
assign db

**Comment by makortel** [comment]
apparently gets fixed with https://github.com/cms-sw/cmssw/pull/34350

**Comment by iarspider** [comment]
@ggovi @makortel Yes, this should be closed in favor of #34350.
